### ISSUE TITLE
2.6 UX: Bring together V2 Cluster Provisioning with Ember-embedded V1

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#2736]
Need a combined cluster list that shows V2 and V1 clusters.
Need to ensure that V1 embedding appears correctly when V2 is turned on and off via the tech preview/beta flag

**Comment by thehejik** [comment]
**Test report:**

TL;DR - it works as described above. I was testing mainly with `legacy` flag turned off (but also tried to turn it on) and I was altering between `RKE/RKE2` switch and `RKE2` feature flag - both ways are offering only the corresponding UIs for provisioning/importing a cluster - Vue and/or Ember. 

* I didn't try to provision anything apart Hosted clusters (AKS, EKS, GKE) to make sure that backend resources are fetched correctly when iframes are used & custom RKE2 single node cluster.
* I was creating credentials and templates for cloud provisioners within cluster definition (not always - sometimes I used existing ones) to make sure the data are propagated correctly into rancher.

I identified following related bugs during my testing:
* NEW https://github.com/rancher/rancher/issues/33343 Problems with loading `RKE1 Configuration` iframes in Dashboard
* EXISTING https://github.com/rancher/dashboard/issues/3145 No easy way for displaying reg. command for existing RKE1 in Dashboard

Create cluster scenario:
- **Hosted Kubernetes - always iframe**: EKS, AKS, GKE (AKS failed for unknown reason - but v2 is ongoing, GKE failed due https://github.com/rancher/rancher/issues/33241, AKS worked)
- **RKE2 node drivers - all Vue**: EC2, AZURE, DO, Linode (no support yet but it is active), vSphere (no env to test - fake credentials not accepted)
- **RKE1 node drivers - all iframe**: EC2, Azure, DO, Linode, vSphere (fake creds accepted but ofc vsphere res not fetched)
- **Custom RKE2 cluster - Vue**: I was able provision custom RKE2 single-node cluster with all roles set
- **Custom RKE1 cluster - iframe**: the reg. command looks ok - but no easy way to get reg. command again due https://github.com/rancher/dashboard/issues/3145

Import existing cluster scenario:
- **Hosted Kubernetes - always iframe**: EKS, AKS, GKE - it offers a list of "cluster To register" when credentials are provided/created in given region
- **Generic - Vue**: kubectl command(s) are showing up after a while in Registration tab

Closing.
